### DISCUSSION TITLE
How to set padding for the entire canvas?

### DISCUSSION THREAD

**Discussion by nikhilweee** [cetz-package/cetz#397] [Q&A] [answered]
The following image follows the code snippet below. Is there a way to set padding for the entire canvas? 

```typst
#cetz.canvas(
  background: luma(240),
  {
    line(...) // trapezium left
    line(...) // trapezium right
  }
)

```

Essentially, I want the background to also fill in the white border shown.

<img width="403" alt="image" src="https://github.com/johannes-wolf/cetz/assets/8774119/72b3d9a2-e66f-4aa5-ab69-33f95390ef80">

**Comment by johannes-wolf** [comment, marked as answer]
If you are on 0.1.2 you could draw a hidden rect: `rect(..your frame, stroke: none, fill: none)` to
force resize the canvas. With 0.2.0 you can put everything in a group and set the groups `padding:` attribute.
